src/test/java/uk/gov/digital/ho/proving/income/validator/CatANonSalariedIncomeValidatorTest.java

```diff
@@ -372,4 +391,15 @@ private void assertExpectedResult(IncomeValidationRequest request, IncomeValidat
         IncomeValidationResult result = validator.validate(request);
         assertThat(result.status()).isEqualTo(expectedStatus);
     }
+
+    private void expectDependants(int dependants) {
```

Comment:
There seems to be a lot of this needed now because we inject the component. That's denfintely better than the way it was but it's almost like the Threshold Calculator is being rewritten for each test. Perhaps we can create a test version of the component with the values hard coded or something?